packages/components/src/components/date-picker/date-picker.jsx

```diff
@@ -251,7 +251,6 @@ DatePicker.defaultProps = {
     date_format: 'YYYY-MM-DD',
     mode: 'date',
     display_format: 'DD MMM YYYY',
-    keep_open: false,
```

Comment:
Why do we need this change?
Do we need to remove it from `trading-date-picker.jsx`

Reply:
Will revert this